### PR TITLE
[DM-3867] The OPC UA gateway no longer fails when running in a Docker container that lacks specific 64-bit system functions

### PR DESCRIPTION
**Pull request by bggov**
Do not merge!

This is a Draft PR. When we have a build number for this fix I'll update it in the release notes and then make it as an official PR.

### CODE REVIEW

File: content/change-logs/device-management/opcua-10-20-change-me-OPC-UA-Gateway-run-on-arm64-without-speciffic-64bist-system-libs.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
+ticket: DM-3867
+version: 10.20.change.me
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/opcua-10-20-change-me-OPC-UA-Gateway-run-on-arm64-without-speciffic-64bist-system-libs.md:15]:
```suggestion
version: 10.20.77.0
```

**Review by Oezge-Akin — APPROVED** [review]

File: content/change-logs/device-management/opcua-10-20-77-0-OPC-UA-Gateway-run-on-arm64-without-speciffic-64bist-system-libs.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
+ticket: DM-3867
+version: 10.20.77.0
+---
+In certain ARM64-based environments, the OPC UA gateway may fail when attempting to call a specific 64-bit system function that might be missing in some base Docker images. This issue has now been resolved. If this function is missing, the service will attempt to call a default system function before failing.
```
Comment (BeateRixen) [inline content/change-logs/device-management/opcua-10-20-77-0-OPC-UA-Gateway-run-on-arm64-without-speciffic-64bist-system-libs.md:17]:
```suggestion
In certain ARM64-based environments, the OPC UA gateway may have failed when attempting to call a specific 64-bit system function that might be missing in some base Docker images. This issue has now been resolved. If this function is missing, the service will attempt to call a default system function instead.
```

File: content/change-logs/device-management/opcua-10-20-77-0-OPC-UA-Gateway-run-on-arm64-without-speciffic-64bist-system-libs.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: 
+title: The OPC UA gateway no longer fails when running in a Docker container that lacks specific 64-bit system functions
```
Comment (BeateRixen) [inline content/change-logs/device-management/opcua-10-20-77-0-OPC-UA-Gateway-run-on-arm64-without-speciffic-64bist-system-libs.md:3]:
```suggestion
title: OPC UA gateway no longer fails when attempting to call a missing 64-bit system function
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@bggov Can this one be merged now as well as it no longer shows the draft mode? I copied over the texts from the approved CD PR.

**Pull request merged**